### PR TITLE
DBZ-1402 Support for JDK 11

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.jboss.org/browse/DBZ-1402

Supersedes https://github.com/debezium/debezium/pull/1039

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -18,7 +18,7 @@ sudo: required
 dist: trusty
 
 jdk:
-  - oraclejdk8
+  - oraclejdk11
```
Comment (gunnarmorling) [inline .travis.yml]: @jpechane, as per the discussion in the original PR I thought we wanted to keep the build running on 8 for the time being?

Comment (gunnarmorling) [reply .travis.yml]: @jpechane ping ^

Comment (jpechane) [reply .travis.yml]: @gunnarmorling Done

**Comment by gunnarmorling** [comment]
LGTM, just awaiting the re-triggered build, will then rebase and apply.

Thanks a lot for your continued effort on this, @brbrown25 and thanks for wrapping it up, @jpechane! Very nice that we now can move the container images to JDK 11.

**Comment by gunnarmorling** [comment]
Applied 🎉 !

**Comment by brbrown25** [comment]
Thanks so much for the opportunity!

Brandon Brown
(240) 506-8335 (m)

**Pull request merged**